src/ui/components/EntryDialogues/DialogSwitchPublic/DialogPublic/AuthorSection/AuthorSection.tsx

```diff
@@ -21,29 +23,32 @@ type Props = {
 };
 
 function AuthorSection(props: Props) {
-    const {className, authorData, disabled, onChange, scope} = props;
+    const {className, authorData, disabled, onChange, scope, validationErrors} = props;
 
     return (
         <div className={b(null, className)}>
             <div className={b('title')}>{i18n('section_author')}</div>
             <div className={b('description')}>
                 {i18n('label_author-description', {subject: i18n(`label_author-subject-${scope}`)})}
             </div>
-            <div className={b('row')}>
-                <div className={b('row-prefix')}>{i18n('label_author-text')}</div>
-                <TextInput
-                    value={authorData.text}
-                    disabled={disabled}
-                    onUpdate={(value) => onChange({text: value})}
-                ></TextInput>
-            </div>
-            <div className={b('row')}>
-                <div className={b('row-prefix')}>{i18n('label_author-link')}</div>
-                <TextInput
-                    value={authorData.link}
-                    disabled={disabled}
-                    onUpdate={(value) => onChange({link: value})}
-                ></TextInput>
+
+            <div className={b('form')}>
+                <FormRow label={i18n('label_author-text')} className={b('form-row')}>
+                    <TextInput
+                        error={validationErrors.text ?? false}
```

Comment:
Do we have any validation errors for this field?

Reply:
Yep, not yet implemented, but i've done validationErrors for both fields in form